boa/src/vm/mod.rs

```diff
@@ -301,7 +301,8 @@ impl Context {
             }
             Opcode::DefInitArg => {
                 let index = self.vm.read::<u32>();
-                let name = self.vm.frame().code.variables[index as usize];
+                let name_str = self.vm.frame().code.variables[index as usize].clone();
+                let name = self.interner_mut().get_or_intern(name_str);
```

Comment:
At some point we'd want to have `variables` be an array of `Sym`s, right?

Reply:
Yes, but it was too much for this first version